src/components/Wines.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from "react";
```

Comment:
We could filter the products instead of creating a new component, but this is fine too.